src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/fakta/BehandlingFaktaService.kt

```diff
@@ -148,7 +148,19 @@ class BehandlingFaktaService(
 
     private fun mapAktivitetForDagligReise(aktivitet: AktivitetDagligReiseAvsnitt?) =
         FaktaAktivtetDagligReise(
-            aktivitet = mapAktivitet(aktivitet?.aktivitet),
+            aktivitet =
+                FaktaAktivtet(
+                    søknadsgrunnlag =
+                        aktivitet?.let {
+                            SøknadsgrunnlagAktivitet(
+                                aktiviteter =
+                                    it.aktiviteter
+                                        ?.map { it.label },
+                                annenAktivitet = it.annenAktivitet,
+                                lønnetAktivitet = null,
```

Comment:
Her stiller vi ikke dette spørsmålet for daglig reise, men heller:
- Hva slags type arbeidsrettet aktivitet går du på?
- Er du lærling, lærekandidat, praksisbrevkandidat eller kandidat for fagbrev på jobb?
- Får du dekket reisen til aktivitetsstedet av arbeidsgiveren din?
- Er du under 25 år hele skoleåret?

Legger til støtte for disse spørsmålene i ny PR